Backpack-Fonts/Scripts/download-relative-fonts.rb

```diff
@@ -12,7 +12,7 @@
     system "git init > /dev/null"
     system "git remote add origin git@github.skyscannertools.net:backpack/bpk-fonts.git > /dev/null"
     system "git config core.sparsecheckout true > /dev/null"
-    system "echo \"iOS\" >> .git/info/sparse-checkout > /dev/null"
+    system "echo \"iOS\" >> .git/info/sparse-checkout"
```

Comment:
Turns out `> /dev/null` was preventing anything being actually added to `sparse-checkout` 🤦